package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "neomad.org",
+  "description": "Neomad is a open-source community project for nomad workers and people willing to _nomadize_ themselves.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "babel": "babel",
```

Comment:
what is this script command for?
What's the purpose of running `npm run babel`?
Same question for `npm run webpack`.

Can you confirm @fuentesloic that `npm run build` compiles the js files ready for production?

It could be interesting to have a `npm run watch` so that developers could easily have their assets rebuilt when changing React components or other webpack items.